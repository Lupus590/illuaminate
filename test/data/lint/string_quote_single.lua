-- config: (lint (only format:string-quote) (quote single))

print("foo", 'foo', [[foo]])
print("'\n\"\'", '"\n\"\'', [[
"']])
