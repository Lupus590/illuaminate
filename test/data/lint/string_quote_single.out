string_quote_single.lua:[3:7-3:11]: String should use single quotes ('\'') [format:string-quote]
   │
 3 │ print("foo", 'foo', [[foo]])
   │       ^^^^^
string_quote_single.lua:[4:7-4:15]: String should use single quotes ('\'') [format:string-quote]
   │
 4 │ print("'\n\"\'", '"\n\"\'', [[
   │       ^^^^^^^^^
No errors and 2 warnings
 -- config: (lint (only format:string-quote) (quote single))
 
-print("foo", 'foo', [[foo]])
-print("'\n\"\'", '"\n\"\'', [[
+print('foo', 'foo', [[foo]])
+print('\'\n\"\'', '"\n\"\'', [[
 "']])
 
